aiobotocore/httpsession.py

```diff
@@ -195,28 +240,58 @@ async def send(self, request):
             # https://github.com/boto/botocore/issues/1255
             headers_['Accept-Encoding'] = 'identity'
 
-            chunked = None
-            if headers_.get('Transfer-Encoding', '').lower() == 'chunked':
-                # aiohttp wants chunking as a param, and not a header
-                headers_.pop('Transfer-Encoding', '')
-                chunked = True
+            content: bytes | str | None = None
+
+            # previously data was wrapped in _IOBaseWrapper
+            # github.com/aio-libs/aiohttp/issues/1907
+            # I haven't researched whether that's relevant with httpx.
```

Comment:
ya silly decision of aiohttp, they took over the stream.  Most likely httpx does the right thing.  I think to get around the sync/async thing we can just make a stream wrapper that hides the relevant methods...I think I did this somewhere...will try to remember